google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -227,6 +228,11 @@ public static CreateUpdateDeleteAdminApiOption validateOnly(Boolean validateOnly
     return new ValidateOnlyOption(validateOnly);
   }
 
+  /** Option to request DirectedRead for ReadOnlyTransaction and SingleUseTransaction. */
```

Comment:
Can we elaborate on what this option does? I have no idea when I should use directed read and when I should not.

Reply:
Added some more documentation.